{"name": "test_02", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_02():\n        print(\"test_2\")\n>       assert 1==2\nE       assert 1 == 2\n\n..\\test_case\\test_case_01.py:12: AssertionError"}, "attachments": [{"name": "stdout", "source": "b49fc1a4-efb7-4eb2-acc1-d2a4cb21402a-attachment.txt", "type": "text/plain"}], "start": 1585014187842, "stop": 1585014187843, "uuid": "611145e7-85a6-48cc-8667-b06c14913e9e", "historyId": "67a9398156540bebfc41a2248395b49b", "testCaseId": "12ca7a6a198dbb9f75ffce63b5a0c13d", "fullName": "test_case.test_case_01#test_02", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_01"}, {"name": "host", "value": "SC-201906191310"}, {"name": "thread", "value": "6524-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_01"}]}